src/test/kotlin/uk/gov/dluhc/printapi/mapper/aed/AnonymousSearchSummaryMapperTest.kt

```diff
@@ -0,0 +1,95 @@
+package uk.gov.dluhc.printapi.mapper.aed
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.any
+import org.mockito.kotlin.given
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.verifyNoInteractions
+import org.mockito.kotlin.verifyNoMoreInteractions
+import uk.gov.dluhc.printapi.mapper.InstantMapper
+import uk.gov.dluhc.printapi.testsupport.testdata.dto.aed.buildAnonymousSearchSummaryDto
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildAnonymousElectorDocumentSummaryEntity
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildAedSearchSummaryApi
+import java.time.OffsetDateTime
+
+@ExtendWith(MockitoExtension::class)
+class AnonymousSearchSummaryMapperTest {
+
+    @Mock
+    private lateinit var instantMapper: InstantMapper
+
+    @InjectMocks
+    private lateinit var mapper: AnonymousSearchSummaryMapperImpl
+
+    @Nested
```

Comment:
Minor ... question over inner classes like this ....
I understand the value inner classes has when there are multiple test methods grouped in the inner class, but in this case we dont ... 🤔 

FWIW I've just ripped out an inner class in the mapper test I am currently working on for this very reason ....